bin/stencil-start

```diff
@@ -2,7 +2,7 @@
 
 var _ = require('lodash');
 var Bs = require('browser-sync').create();
-var Cycles = require('../lib/cycles');
+var Cycles = require('../lib/Cycles');
```

Comment:
Can you rename the file to `cycles.js` and test it, that will put it more within standards.